src/utils/authorization.ts

```diff
@@ -4,3 +4,5 @@ const publisherAffiliations = [Affiliation.EMPLOYEE, Affiliation.STAFF, Affiliat
 
 export const checkIfPublisher = (user: User) =>
   user.isLoggedIn && user.affiliations.some(userAffiliation => publisherAffiliations.includes(userAffiliation));
+
+export const checkIfAppAdmin = (user: User) => user.isLoggedIn && user.email.endsWith('@unit.no');
```

Comment:
Måtte denne aldri havne i prod :-) Heldigvis stoler vi på backend

Reply:
SHIP IT